### PR TITLE
Simple field-less enums

### LINKED ISSUE #238: Enum support
**Is your feature request related to a problem? Please describe.**
Trying to generate an enum results in `thread 'main' panicked at 'parse_type failed for ty=EnumName'`

**Describe the solution you'd like**
Rusts enums are super powerful (and I would say impossible to fully support) but it would be nice to at least support the most basic enum types 

```
pub enum StartFrom {
    Beginning,
    End,
}
```

**Describe alternatives you've considered**
Passing Strings now is the workaround for this limitation

**Additional context**
-

### PR DESCRIPTION
**Pull request by Desdaemon**
Resolves #238.

### CODE REVIEW

File: frb_codegen/src/generator_rust.rs [hunk 1]
```diff
@@ -397,9 +397,11 @@ impl Generator {
             };
             vec.into_iter().map(Wire2Api::wire2api).collect()"
                 .into(),
-            Boxed(_) => "let wrap = unsafe { support::box_from_leak_ptr(self) };
-            (*wrap).wire2api().into()"
-                .into(),
+            Boxed(inner) => match inner.as_ref() {
+                ApiTypeBoxed { inner: ApiType::Primitive(_), exist_in_real_api: false } => "unsafe { *support::box_from_leak_ptr(self) }".into(),
```
Comment (Desdaemon) [inline frb_codegen/src/generator_rust.rs:401]: Fixes `*mut i32 -> i32` and `*mut i32 -> Box<i32>` wire2api functions

**Comment by fzyzcjy** [comment]
Thank you!

**Comment by fzyzcjy** [comment]
Hmm there must be something wrong with ios CI...

**Comment by fzyzcjy** [comment]
let me fix it first before merging this pr

**Comment by fzyzcjy** [comment]
running: https://github.com/fzyzcjy/flutter_rust_bridge/runs/4439308730?check_suite_focus=true

**Comment by fzyzcjy** [comment]
🎉

**Comment by fzyzcjy** [comment]
@Desdaemon Hi, I am open-sourcing a [new library](https://github.com/fzyzcjy/flutter_modern_test) inspired by [Cypress](https://github.com/cypress-io/cypress) which has 35k stars. Are you interested in being a co-author? 😄

**Comment by Desdaemon** [comment]
@fzyzcjy Thanks! I am quite inexperienced with testing in general, but otherwise would love to work on a project like this.

**Comment by fzyzcjy** [comment]
@Desdaemon You are welcome!

**Pull request merged**